lambda_functions/vetext_incoming_forwarder_lambda/vetext_incoming_forwarder_lambda.py

```diff
@@ -213,27 +222,50 @@ def read_from_ssm(key: str) -> str:
 
 
 def make_vetext_request(request_body):
-    ssm_path = os.getenv('vetext_api_auth_ssm_path')
-    if ssm_path is None:
-        logger.error('Unable to retrieve vetext_api_auth_ssm_path from env variables')
-        return None
-
-    domain = os.getenv('vetext_api_endpoint_domain')
-    if domain is None:
-        logger.error('Unable to retrieve vetext_api_endpoint_domain from env variables')
-        return None
-
-    path = os.getenv('vetext_api_endpoint_path')
-    if path is None:
-        logger.error('Unable to retrieve vetext_api_endpoint_path from env variables')
-        return None
+    endpoint = request_body.get('path', '/twoway/vettext')
```

Comment:
Did we want to make these paths python Constants

Reply:
Yes they should be, this would be easier to add more if needed then, or if they ever change.

Reply:
Ok, I added constants for the twilio paths